onnx/checker.h

```diff
@@ -1,7 +1,11 @@
 #pragma once
 
 #include <stdexcept>
+#ifdef ONNX_ML
```

Comment:
Maybe move this `ifdef` chunk into a separated file so other files can use it as a proxy to the correct pb.h file?

Reply:
Great suggestion! Will do it and send new iteration.